src/main/java/com/epam/prd/jjop/helpers/objects/NullWonder.java

```diff
@@ -0,0 +1,79 @@
+package com.epam.prd.jjop.helpers.objects;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.Objects;
+
+/**
+ * Description of {@link java.util.Objects} regarding null problem solving.
+ * <p>
+ * This class show examples of all methods to check for {@code null}.
+ * Description of the differences between these methods can be found here.
+ * </p>
+ *
+ * @author Krzysztof Dolański
+ * Objects class from java.util @since 1.7
+ */
+
+public class NullWonder {
+
+
+    public static void main(String[] args) {
+
+        String assumeNull = null;
+
+        /**
+         *
+         * Using null object as parameter in this method will produce 0 as return.
+         *
+         * <p>
+         * This method under the hood is using Arrays.hashCode
+         * </p>
+         * <p>
+         * WARNING - in case of passing more than one null parameter return value WILL NOT be 0 because of implementation of Arrays.hashCode()
+         * {@link java.util.Arrays.hashCode()};
```

Comment:
move that to end of line 33 (replace not linked version).

Reply:
Do you mean that line with {@link java.util Arrays.hashCode()} should be in line 33 in the end of that line?
I sliced this sentence for better reading (like grey vertical line show).
Not linked version replaced.

Reply:
Well at the end of lien you have `Arrays.hashCode() {@link java.util.Arrays#hashCode()}`.

Just cut off `Arrays.hashCode()` and leave the thing in `{@link}`. What there is currently is unnecessary repetition.

Reply:
:) done